src/upgrade/nc_upgrade_scripts/1.0.0/config_dir_restructure.js

```diff
@@ -248,21 +252,21 @@ async function create_account_access_keys_index_if_missing(config_fs, account_up
  * 2. iterates all old accounts to a hidden directory
  * 3. deletes the accounts/ directory
  * // TODO - consider removing the accounts in the future, currently we decide to not delete old accounts
- * @param {nb.NativeFSContext} fs_context 
  * @param {import('../../../sdk/config_fs').ConfigFS} config_fs 
  * @param {String[]} old_account_names 
  * @param {String} from_version 
  * @param {*} dbg 
  * @returns {Promise<Void>}
  */
-async function move_old_accounts_dir(fs_context, config_fs, old_account_names, from_version, dbg) {
+async function move_old_accounts_dir(config_fs, old_account_names, from_version, dbg) {
```

Comment:
Why `old_account_names`?
I think it might be confusing, as the `old` refers to the account path (not the account name)
```suggestion
async function move_old_accounts_dir(config_fs, account_names, from_version, dbg) {
```